tardis/plasma/properties/rate_matrix_index.py

```diff
@@ -14,24 +14,39 @@ def __init__(self, plasma_parent, nlte_ionization_species=0):
         self.nlte_ionization_species = nlte_ionization_species
 
     def calculate(self, levels, nlte_ionization_species):
+        """Generates rate_matrix_index using levels and changing the last index(level) to "lte_ion" if that ion_number is treated in LTE, "nlte_ion" for NLTE ionizatin and keeps the levels for the rest.
```

Comment:
Should be on multiple lines for a long string

Reply:
Done